keyboards/keebzdotnet/wazowski/rules.mk

```diff
@@ -14,11 +14,12 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+VIA_ENABLE = yes
```

Comment:
```suggestion
```
This should not be enabled at the keyboard level.

Reply:
where should this be enabled at?

Reply:
In the `via` keymap's rules.mk - but you're already done that. It just shouldn't be enabled for *all* keymaps.